runtime/runtime.go

```diff
@@ -137,14 +138,15 @@ func (a *Applet) Run(config map[string]string, initializers ...ThreadInitializer
 			var starlarkVal starlark.Value
 			starlarkVal = starlark.String(v)
 
-			if a.schema != nil {
+			if strings.HasPrefix(k, "$") {
+				// this a special field like "$tz". no need to check the schema
+			} else if a.schema != nil {
 				// app has a schema, so we can provide strongly typed config values
 				field := a.schema.Field(k)
 
 				if field == nil {
 					// we have a value, but it's not part of the app's schema.
 					// drop it entirely.
```

Comment:
We may want to consider allowing this for now and adding a log line - we can follow up in a few days to remove it? Just would hate for there to be a field that we used to pass that we forgot about. Taking a look now to see if there would be any such case just to double check

Reply:
I think this has to stay until we serialize config values to the correct type in storage (currently all config values are serialized to string in actual storage).

Imagine the following sequence of events:

1. Your app has a schema with a toggle/boolean field.
2. Users install it, we save that config value. When your app is run, we pass `True` or `False`.
3. You remove the field from your schema.

Since our serialization is not type-specific, we now no longer have the data to know whether the field was string or bool. So the only safe thing to do is not pass any value.

Reply:
Sorry, misunderstood your comment. Prepping new PR to do as you suggest.